specification/billing/resource-manager/Microsoft.Billing/preview/2020-12-15-privatepreview/subscription.json

```diff
@@ -703,6 +854,48 @@
         }
       }
     },
+    "EnrollmentAccountSubscriptionDetails": {
+      "description": "The billing properties that can be modified. available only for the Enterprise Agreement Type.",
+      "properties": {
+        "subscriptionEnrollmentAccountStatus": {
```

Comment:
why not just "enrollmentAccountStatus"?

Reply:
enrollmentAccountStatus is a different status in EA. it refers to whether an account is active/pending or inactive. This one is more like, whether a subscription is active/canceled or transferred for an account. Its basically a relationship status between Account and Subscription Status.